### ISSUE TITLE
Dialog "File Sync Errors" not accessible with screenreader

### ISSUE THREAD

**Issue by heull001** [haiwen/seafile-client#1461]
The Window "File Sync Errors" is not accessible with screenreaders, I only get the title of the window.

OS: Windows 22H2 (Build 19045.2965)
Client: 9.0.2 (occured in earlier versions too)
Screenreader:
- JAWS 2022.2211.7
- NVDA 2023.1

If you like to test, get NVDA from https://www.nvaccess.org/download/

**Comment by killing** [comment]
This is fixed since 9.0.5 version.

**Comment by heull001** [comment]
JAWS still reads nothing than the title. NVDA is able to read the content of the whole window, but this is not very helpfull, because every entry is read without break. A list to navigate with the arrow keys would be much better.

So could you reopen please @killing ?
